src/index.js

```diff
@@ -1,4 +1,6 @@
 require("dotenv").config();
+
+//Configure and Initialize Apollo Server
```

Comment:
remove comments. Comments should only be included for things that are hacky or hard to understand

Reply:
Took them out.